src/utils/oauth/GithubProvider.ts

```diff
@@ -19,13 +19,13 @@ let details = providerConfig.details.deploy;
 const GithubProvider: IProvider<boolean> = {
   /**
    * Builds authorization url
-   * 
+   *
    * @returns {string} The builded authorization url.
    * @description Build the authorization url for the OAuth-Request, the values are obtained from the config file.
```

Comment:
Remove dot.

Reply:
No looks good. This will be removed in the ongoing documentation PR.